src/zenml/console.py

```diff
@@ -16,13 +16,13 @@
 from rich.console import Console
 from rich.theme import Theme
 
-zenml_custom_theme = Theme(
-    {
-        "info": "dim cyan",
-        "warning": "yellow",
-        "danger": "bold red",
-        "title": "bold cyan underline",
-        "error": "red",
-    }
-)
+zenml_style_defaults = {
+    "info": "dim cyan",
+    "warning": "yellow",
+    "danger": "bold red",
+    "title": "bold cyan underline",
+    "error": "red",
+}
+
+zenml_custom_theme = Theme(zenml_style_defaults)
```

Comment:
Is this theme something that `rich` uses internally as well, or can we remove it as it's now not used by our `cli_utils.declare/warning` functions?
And if we can't, should we somehow read the colors for the `cli_utils` from here in order to have just one place where we define them?

Reply:
The problem is that when someone imports our `console`, they'll want to get these presets built-in, so that's why we need to keep it.

Also, the other problem is that these aren't just colours, so it's not just a simple matter of parsing the k:v pairs. The alternative would be to store these as `Style` objects, which might make sense?

Reply:
Can you have `Style` objects in a rich `Theme`? If yes I feel like that would be a good compromise, and in `cli_utils.warning` we could get that style from the dict and modify it. But if not possible or too much work, feel free to ignore.

Reply:
Themes are basically mappings of names of styles with their corresponding `Style` objects. I've made changes accordingly.